go/fn/object.go

```diff
@@ -666,6 +675,14 @@ func (o KubeObjects) Less(i, j int) bool {
 	return idStrI < idStrJ
 }
 
+func (o *KubeObject) IsDummy() bool {
```

Comment:
IMO `IsDummy` is not an intuitive name. `IsNil` feels more intuitive name.
Please add godoc.